src/main/java/com/tragent/inventory/model/Category.java

```diff
@@ -0,0 +1,56 @@
+package com.tragent.inventory.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "category")
+public class Category {
```

Comment:
She can link this to the products(that is do it as a subresource) or vice versa once they have implemented the products endpoint